doc/sphinx/source/algorithms.rst

```diff
@@ -92,14 +92,8 @@ These all assume that an object of :class:`.GraphDataScience` is available as `g
 
     Constructs a random subgraph based on random walks with restarts.
 
-.. py:function:: gds.graph.sample.cnarw(graph_name: str, from_G: Graph, **config: Any)
-    -> Tuple[Graph, "Series[Any]"]
-
-    Constructs a random subgraph based on common-neighbour-aware random walks.
-
-.. py:function:: gds.graph.sample.cnarw.estimate(G: Graph, **config: Any) -> "Series[Any]"
-
-    Returns an estimation of the memory consumption for that procedure.
+.. deprecated:: 2.4.0
+   Since GDS server version 2.4.0 you should use the endpoint :func:`gds.graph.sample.rwr` instead.
```

Comment:
I believe this is correct use of this sphinx directive. But it does render kind of funny in our custom theme @nvitucci @recrwplay

Reply:
Yes. the theme inherits the behaviour from the main docs. When a div has `class="deprecated`, we use the `::after` pseudo-class on a `<span>` inside the `<div>` to show a deprecated label. We can modify the css in the Sphinx theme to override this.

Reply:
Ah ok :) That sounds good :+1: Is that something you want to do? Or do you need help from us?